NuKeeper/NuKeeper.csproj

```diff
@@ -1,13 +1,18 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFrameworks>netcoreapp1.1;Net462</TargetFrameworks>
     <RootNamespace>NuKeeper</RootNamespace>
     <AssemblyName>NuKeeper</AssemblyName>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="EasyConfig.Net.Core" Version="1.0.6" />
     <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
     <PackageReference Include="NuGet.Protocol.Core.v3" Version="4.0.0" />
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp1.1' ">    
+	<PackageReference Include="EasyConfig.Net.Core" Version="1.0.6" />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'Net462' ">    
+	<PackageReference Include="EasyConfig.Net" Version="1.1.22" />
```

Comment:
agreed that it's messy.
I think that what has to happen is that the easyconfig lib has to also have `<TargetFrameworks>netstandard1.6;Net452</TargetFrameworks>` or similar in the csproj.
i.e. make the nuget package contain build artefacts for both platforms.